src/utils/testHelper.ts

```diff
@@ -0,0 +1,11 @@
+import { ResponseError } from '../pinecone-generated-ts-fetch';
```

Comment:
**question:** Can this file be removed? Doesn't seem like it's actually being used.

Reply:
Yeah, I think it can be removed. I had created a new test-helper in here but later I ended up deleting all the unit tests that were using it. 😂